make.sh

```diff
@@ -101,8 +94,6 @@ rule() {
       tar xf master.tar.gz && rm master.tar.gz
       rm -rf linux-headers && mv linux-headers-master linux-headers
       for n in $(compgen -c gcc- | sed 's/gcc-//'); do if [ $n != 5 ]; then cp -n linux-headers/include/linux/compiler-gcc{5,$n}.h; fi; done
-    ;; lock)
-      opam lock
```

Comment:
Why is this outdated?

Reply:
Because it doesn't work and requires manual attention. Documentation describes how to do it properly: https://github.com/goblint/analyzer/blob/196e82f5a942ede12a6fac00dd9eb5365386c038/docs/developer-guide/releasing.md#L18-L21